core/CliMulti.php

```diff
@@ -304,4 +338,17 @@ private function requestUrls(array $piwikUrls)
         return $results;
     }
 
+    public static function getSuperUserTokenAuths()
```

Comment:
is it necessary that it is public? (in the diff it seems to be called only from this class)